src/ui/interface.js

```diff
@@ -96,6 +103,61 @@ module.exports = function Interface(options) {
       return true;
     });
 
+    $("#local-video-activate").click(function () {
+      if(isCamera){ 
+        localStream.getVideoTracks()[0].stop();  
+      }             
+      isCamera = false;
+      if(!isVideo){ //Prevent Creation of Duplicate video Elements
+        localVideo = document.createElement('video');
+        localVideo.setAttribute("src", "../video.mp4");
+        localVideo.setAttribute("id", "localVideo");
+        localVideo.style.display = "none"               
+        localVideo.style.width = "100px";   
+        localVideo.style.height = "50px";    
+        document.getElementById("video-container").appendChild(localVideo);
```

Comment:
It looks like everything is happening in the DOM here. Did you get a particular error you can share?

Reply:
https://github.com/publiclab/infragram/blob/Forchapeatl-patch-1/src/Infragram.js#L38-L39


Sorry about the mix up @jywarren . I meant that HD videos have a slower frame rate transition on the Infragram canvas.

For example,  HD [video](https://github.com/bower-media-samples/big-buck-bunny-1080p-30s/blob/master/video.mp4) makes better frame transitions with the below settings
```
    if (options.processor.type == "webgl") interval = 1;
    else interval = 15;
```

Reply:
I see, ok! What did you think about making this driven by frame conversion completion instead of interval?